### DISCUSSION TITLE
Support without Connect?

### DISCUSSION THREAD

**Discussion by reparteeanimal** [homebridge-plugins/homebridge-august#45] [Q&A]
Perhaps this is entirely naive, but does this script (or any others) replace the need for a Connect hub? Or is that just an impossibility? I own a (non-Wifi) Smart Lock (non-pro) only, and I'm not sure how to find what generation but was purchased at least 3 years ago.

**Comment by donavanbecker** [comment]
It doesn't yet. Will look into BLE support, but can't promise anything, and would welcome others helps on BLE

**Comment by marvinjoa** [comment]
> It doesn't yet. Will look into BLE support, but can't promise anything, and would welcome others helps on BLE

Do you have any updates on BLE integration? I don't know how to code. I know https://github.com/bdraco/yalexs-ble can connect via BLE, this repository is also used for HomeAssistant. But it's written in Python and not JS, so HomeBridge integration is not easy as the code would need to be ported first.

**Reply by donavanbecker** [reply]
no update.
